doc/tutorial/conv_arithmetic.txt

```diff
@@ -0,0 +1,180 @@
+.. _conv_arithmetic:
+
+======================
+Convolution arithmetic
+======================
+
+The relationship between a convolution operation's input shape, kernel size,
+stride, padding and its output shape can be confusing at times.
+
+This tutorial aims at providing an intuitive understanding of how the different
+convolution parameters influence the shape of the output it produces.
+
+Refresher: discrete convolution
+===============================
+
+.. todo::
+
+    WRITEME
+
+Here are the factors determining the shape of a convolution's output size along
+a given axis (height or width):
+
+* **Input size**: length of the input axis.
+* **Kernel size**: length of the kernel.
+* **Stride**: distance between two consecutive positions of the kernel.
```

Comment:
It's a little verbose, but just to be sure I'd add "along the given axis" here too